### PR TITLE
Updated docstring

### LINKED ISSUE #11710: Clip_vector_norm
Reformat general https://github.com/unifyai/ivy/issues/791

### PR DESCRIPTION
**Pull request by chetna-mittal**
Close #11710

### CODE REVIEW

**Comment by chetna-mittal** [comment]
Please let me know if there are any suggestions or changes needed?

**Comment by chetna-mittal** [comment]
@fspyridakos Have synced my branch with Ivy

**Comment by fspyridakos** [comment]
Can you sync once again, there were some further changes again, apologies for that

**Comment by chetna-mittal** [comment]
> Can you sync once again, there were some further changes again, apologies for that

I have synced again, could you review please

**Comment by fspyridakos** [comment]
Are you sure? The branch you want to merge doesn't seem to have been updated: https://github.com/chetna-mittal/ivy/tree/Clip_vector_norm

**Comment by chetna-mittal** [comment]
> Are you sure? The branch you want to merge doesn't seem to have been updated: https://github.com/chetna-mittal/ivy/tree/Clip_vector_norm

Could you check now

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -860,24 +860,25 @@ def clip_vector_norm(
...
+        The p-value for computing the p-norm. Default is 2.
     out
         optional output array, for writing the result to. It must have a shape that the
         inputs broadcast to.
 
     Returns
     -------
     ret
-        An array with the vector norm downscaled to the max norm if needed.
+        An array with the matrix norm downscaled to the max norm if needed.
```
Comment (fspyridakos) [inline ivy/functional/ivy/general.py:881]: this should remain as vector

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Can you also include the changes to the equivalent array and container instance methods?

**Comment by fspyridakos** [comment]
Thanks for the update, but I was reffering to the `clip_vector_norm` methods found in `ivy/array/general` and `ivy/container/general`, could you update those with the same changes?

**Comment by chetna-mittal** [comment]
> Thanks for the update, but I was reffering to the `clip_vector_norm` methods found in `ivy/array/general` and `ivy/container/general`, could you update those with the same changes?

I have updated those changes as well

**Comment by fspyridakos** [comment]
Looks good now!

**Pull request merged**